docs/sles-installation-guide.md

```diff
@@ -0,0 +1,80 @@
+# Installing Intel® Clear Containers 3.0 on SLES* version 12 SP1
+
+This document _only_ covers installing on a
+[SLES](https://www.suse.com/products/server/) 12 SP1 system. Although,
+this guide could be used with more recent Service Packs, it has not been
+tested.
+
+Note:
+
+If you are installing on a system that already has Clear Containers 2.x
+installed, first read [the upgrading document](upgrading.md).
+
+## Required Setup
+
+The installation requires the current user to run the `sudo` command
+without specifying a password. Verify this with the following commands:
+
+```
+$ su -
+# echo "$some_user ALL=(ALL:ALL) NOPASSWD: ALL" | (EDITOR="tee -a" visudo)
+$ exit
+
+```
+
+## Installation steps
+
+1. Ensure the system packages are up-to-date:
+
+```
+$ sudo zypper update
```

Comment:
It might be an idea to change all occurrences of `sudo` to `sudo -E` for consistency with the other install guides. I can see that we don't specify that in the CentOS guide, but we do for Ubuntu + Fedora.